test/src/graph/nodes/Ros2PublishRadarScanNodeTest.cpp

```diff
@@ -0,0 +1,73 @@
+#include <helpers/commonHelpers.hpp>
+#include <helpers/sceneHelpers.hpp>
+#include <helpers/graphHelpers.hpp>
+#include <helpers/testPointCloud.hpp>
+
+#include <graph/NodesRos2.hpp>
+
+#include <rclcpp/rclcpp.hpp>
+
+// TODO: replace with API calls
+#include <api/apiCommon.hpp>
+
+TEST(Ros2PublishRadarScanNodeTest, should_receive_sent_data)
```

Comment:
```suggestion
class Ros2PublishRadarScanNodeTest : public RGLTest {};

TEST_F(Ros2PublishRadarScanNodeTest, should_receive_sent_data)
```
You should define `Ros2PublishRadarScanNodeTest` class and derive it from `RGLTest` to set the logging configuration at the test startup and call `rgl_cleanup` at the test exit.
Also, it should be a Test Fixture.